### PR TITLE
Create survey button, add nav helper, add alert for new talk button

### PR DESCRIPTION
**Pull request by tmclean85**
Push survey scene from popup, weekend work

### CODE REVIEW

File: js/navigation/NavigationLayout.ios.js [hunk 1]
```diff
@@ -74,7 +74,9 @@ class NavigationLayout extends Component {
           title="new talk"
           renderTitle={this.renderTitle}
           renderIcon={() => <Image source={require('../assets/icons/newtalk_icon.png')} style={styles.navIconIos} />}
```
Comment (codedavinci) [inline js/navigation/NavigationLayout.ios.js]: Don't` require` , `import` instead

File: js/navigation/NavigationLayout.ios.js [hunk 2]
```diff
@@ -102,7 +104,7 @@ class NavigationLayout extends Component {
           renderIcon={() => <Image source={require('../assets/icons/more_icon.png')} style={styles.navIconIos} />}
```
Comment (codedavinci) [inline js/navigation/NavigationLayout.ios.js]: Don'trequire , import instead

**Review by codedavinci — COMMENTED** [review]
To be adjusted

**Comment by tmclean85** [comment]
Swapped image require for imports.

**Review by codedavinci — DISMISSED** [review]
LRGTM !

**Review by codedavinci — APPROVED** [review]
LRGTM !

**Pull request merged**